status/backend.json

```diff
@@ -169,6 +169,14 @@
         "it-IT": "",
         "en-EN": "."
       }
+    },
+    "favourite_language": {
+      "is_visible": false,
+      "level": "normal",
+      "message": {
+        "it-IT": "",
+        "en-EN": ""
```

Comment:
@debiff does this model support German as well?

Reply:
Nope. Remote locales are a separated world from app locales.
They will be support German too in the near future.
In this case the app will shown the message in italian (as a default fallback for the app)

Reply:
The idea here is to include a link to contribute improving the translation—ideally to the "create new issue" page on github